MonoGame.Framework/Input/GamePadCapabilities.cs

```diff
@@ -46,6 +46,10 @@ public struct GamePadCapabilities
     {
         public bool IsConnected { get; internal set; }
 
+        public string DisplayName { get; internal set; }
+
+        public string Id { get; set; }

```

Comment:
Another thing... lets add some docs to these two new properties.

They should clearly state that this is a MonoGame extension.  Also that these strings can be `null` if the platform doesn't support identification of controllers (which is true of all our platforms other than SDL today).